STYLEGUIDE.md

```diff
@@ -374,32 +338,6 @@ object in parentheses rather than using an explicit return:
 }
 ```
 
-### Object / Array iterations, transformations and operations
```

Comment:
I think since we now have `for of` loops we should not be that strictly discouraging usage of `for` anymore. It pretty much boils just down to: please write readable code, sometimes functional might be the better way, sometimes a library, sometimes a `for of` loop. So I think this rule outlived itself.